HPXMLtoOpenStudio/measure.rb

```diff
@@ -3412,12 +3421,12 @@ def self.set_surface_interior(model, spaces, surface, interior_adjacent_to)
     end
   end
 
-  def self.set_surface_exterior(model, spaces, surface, exterior_adjacent_to)
+  def self.set_surface_exterior(model, spaces, surface, exterior_adjacent_to, interior_adjacent_to = nil)
     if exterior_adjacent_to == HPXML::LocationOutside
       surface.setOutsideBoundaryCondition('Outdoors')
     elsif exterior_adjacent_to == HPXML::LocationGround
       surface.setOutsideBoundaryCondition('Foundation')
-    elsif exterior_adjacent_to == HPXML::LocationOtherHousingUnit
+    elsif [HPXML::LocationOtherHousingUnit, interior_adjacent_to].include? exterior_adjacent_to
```

Comment:
👍

Reply:
Actually, this would probably miss the case of a FrameFloor between living space and conditioned basement. How about adding a `is_adiabatic` method (similar to `is_interior` and `is_exterior` methods) to the HPXML class that checks for InteriorAdjacentTo=ExteriorAdjacentTo as well as this conditioned basement scenario?